ALUM/ALUM/Views/SignUpPage/SignUpPageView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SignUpPageView.swift
+//  ALUM
+//
+//  Created by Neelam Gurnani on 2/17/23.
+//
+
+import SwiftUI
+
+struct SignUpPageView: View {
+
+    @StateObject private var viewModel = SignUpPageViewModel()
+
+    var body: some View {
+        NavigationView {
```

Comment:
note - Having the NavigationView for the signup flow is fine for now but in future when we have more screens ready, we can just have a `NavigationView` at the top level in our app, maybe ALUMApp.